faq/storage-management.md

```diff
@@ -0,0 +1,28 @@
+---
+description: This article outlines the most frequently asked questions about storage management in Syskit Point.
+---
+
+# Storage Management in Syskit Point FAQ
+
+
+## What is the difference between Microsoft 365 and Syskit Point when calculating the used SharePoint Online storage space?
+
+SharePoint Online **used storage** is visible inside the SharePoint Admin center; Microsoft refreshes this data periodically.
+
+Syskit Point calculates used storage as a sum of the storage used by each site collection (alive or soft deleted).
+
+Depending on the size of your environment, it takes some time to refresh the data for all sites. 
+
+
+## How often is total storage checked and updated/synced by Syskit Point?
+
+This depends on the size of your environment, as the sync happens gradually. 
+
+**Smaller tenants are synced every 24 hours. With larger tenants, the sync can take up to 7 days.**
+
+If a site is not yet synced and you want to have the current information instantly, you can manually sync your tenant. 
```

Comment:
ovaj zadnji dio promijeniti. ne možemo syncati tenant. pretpostavljam da tu trebamo napisati:
If a site is not yet synced and you want to have the current information instantly, you can manually sync the site from the Site details screen. ili tome slično